src/node_crypto.cc

```diff
@@ -4878,6 +4878,27 @@ void GetHashes(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void GetCurves(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Local<Array> arr = Array::New(env->isolate());
+  size_t len = EC_get_builtin_curves(NULL, 0);
```

Comment:
Perhaps `num_curves` is a better name than `len` and maybe make it `const`.

If you move the `Array::New()` call down a few lines, you can initialize it more efficiently:

```c++
Local<Array> arr = Array::New(env->isolate(), num_curves);
```